WalletWasabi.Fluent/ViewModels/Wallets/Send/SendViewModel.cs

```diff
@@ -350,7 +355,7 @@ protected override void OnNavigatedFrom(bool isInHistory)
 
 		if (!isInHistory && _coinJoinManager is { } coinJoinManager)
 		{
-			coinJoinManager.WalletLeftSendWorkflow(_wallet.WalletName);
+			Task.Run(async () => await coinJoinManager.WalletLeftSendWorkflowAsync(_wallet));
```

Comment:
This is an abandoned task.

Reply:
There will be an abandoned task somewhere, bc i'm calling an `async` func from `sync` context.

I will try to minimize the abandoned task's job.

Do you happen to know a better solution?